package.json

```diff
@@ -12,14 +12,13 @@
     "clean": "node scripts/clean.js",
     "build:ts": "tsc",
     "build:gitignore": "node scripts/update_gitignore_for_tsc_output.js run",
-    "format": "prettier --write \"{lib,bin,scripts,test}/**/*.js\"",
-    "format:check-difference": "prettier --list-different \"{lib,bin,scripts,test}/**/*.js\"",
+    "format": "prettier $([ ! $CI ] && echo '--write') --list-different .",
     "debug:test": "mocha --inspect-brk --exit -t 0 test/all-tests-suite.js",
     "debug:tape": "node --inspect-brk test/tape/index.js",
     "coveralls": "nyc report --reporter=lcov",
-    "lint": "eslint \"lib/**/*.js\" \"test/**/*.js\" \"bin/**/*.js\"",
+    "lint": "eslint --cache $([ ! $CI ] && echo '--fix') '**/*.js'",
```

Comment:
I don't think echo is cross-platform and is likely not work correctly on Windows. Can you add separate command specifically for CI?

Reply:
Can do 👍

Reply:
Done.